lib/WebService/GrowthBook.pm

```diff
@@ -45,11 +50,39 @@ WebService::GrowthBook - sdk of growthbook
 # singletons
 
 class WebService::GrowthBook {
+    field $enabled :param //= 1;
     field $url :param //= 'https://cdn.growthbook.io';
-    field $client_key :param;
+    field $client_key :param //= "";
     field $features :param //= {};
+    field $attributes :param :reader :writer //= {};
     field $cache_ttl :param //= 60;
+    field $user :param //= {};
+    field $forced_variations :param //= {};
+    field $overrides :param //= {};
+    field $sticky_bucket_service :param //= undef;
+    field $groups :param //= {};
+    field $qa_mode :param //= 0;
+    field $on_experiment_viewed :param //= undef;
+    field $tracking_callback :param //= undef;
+
     field $cache //= WebService::GrowthBook::InMemoryFeatureCache->singleton;
+    field $sticky_bucket_assignment_docs //= {};
+    field $tracked = {};
+    field $assigned = {};
+    field $subscriptions = [];
+
+    sub BUILDARGS{
+        my ($class, %args) = @_;
+        adjust_args_camel_to_snake(\%args);
```

Comment:
just curios why do we convert camel_case arguments to snake_case ?

Reply:
because the data loaded from growthbook json file is camel_case, but the perl style is snake_case.